_inc/client/at-a-glance/akismet.jsx

```diff
@@ -33,6 +34,29 @@ class DashAkismet extends Component {
 		isDevMode: '',
 	};
 
+	constructor() {
+		super( ...arguments );
+
+		this.trackInstallClick = this.trackInstallClick.bind( this );
+		this.trackActivateClick = this.trackActivateClick.bind( this );
+	}
+
+	trackInstallClick() {
```

Comment:
What do you think about switching to using arrow functions here instead of using `bind` in the constructor?

Reply:
We don't actually need the binding as those methods don't really need `this` scope.